common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -82,10 +77,14 @@
 import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME;
 import static com.microsoft.identity.common.adal.internal.AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME;
 
-
 public class StorageHelper implements IStorageHelper {
     private static final String TAG = "StorageHelper";
 
+    /**
+     * A flag to turn on/off keystore encryption on Broker apps.
+     */
+    public static final boolean SHOULD_ENCRYPT_BROKER_VALUES_WITH_KEYSTORE_ENCRYPTED_KEY = false;
```

Comment:
nit : not a constant so uppercase not required, use variable name like : `sShouldEncryptWithKeyStoreKey`